scheds/rust/scx_layered/src/bpf/intf.h

```diff
@@ -30,6 +30,7 @@ enum consts {
 	MAX_TASKS		= 131072,
 	MAX_PATH		= 4096,
 	MAX_NUMA_NODES		= 64,
+	MAX_CONTAINERS		= 64,
```

Comment:
Can we use 'cpuset` / `CPUSET` consistently instead of using both `container` and `cpuset` and document the implemented behavior in a comment?